### DISCUSSION TITLE
Input file for OpenFast wake simulation

### DISCUSSION THREAD

**Discussion by danielemeschino** [OpenFAST/openfast#2322] [Q&A]
Hi, I want to simulate the wake of my turbine, for which I have created the blade design on QBlade. From QBlade I have all the airfoils, polars, chord-twist distribution, turbine-simulation (power, Cp, Ct curves). Now my question is: how can I create the input file for the wake simulation on OpenFast? I have tried to read the documentation or to see some videos tutorial, but it seems very difficult to understand something, someone can help me?
Thanks

**Comment by jjonkman** [comment]
Dear @danielemeschino,

By OpenFAST wake simulation, are you referring to the free-vortex wake model implemented in the OLAF submodel of AeroDyn?  Do you want aero-elastic coupling, or will this be an aerodynamic-only simulation?  For the later, you can use the standalone driver of AeroDyn rather than AeroDyn coupled within OpenFAST.  The aerodynamic-related inputs to AeroDyn and OLAF should be quite similar to those of QBlade, although the input file formatting is likely a bit different.  AeroDyn and OLAF, including their input files, are documented within OpenFAST readthedocs: https://openfast.readthedocs.io/en/main/source/user/aerodyn/index.html and https://openfast.readthedocs.io/en/main/source/user/aerodyn-olaf/index.html.

Best regards,
